cmd/neofs-node/config.go

```diff
@@ -522,7 +522,7 @@ type cfgReputation struct {
 	localTrustCtrl *trustcontroller.Controller
 }
 
-var persistateSideChainLastBlockKey = []byte("side_chain_last_processed_block")
+var persistateFSChainLastBlockKey = []byte("fs_chain_last_processed_block")
```

Comment:
This is an incompatible change, we either need to keep the value or provide migration.